config/image.php

```diff
@@ -15,6 +15,6 @@
     |
     */
 
-    'driver' => env('IMAGE_DRIVER', 'gd'),
+    'driver' => env('IMAGE_DRIVER', 'imagick'),
```

Comment:
This is a breaking change, I can't merge this PR without breaking instances that rely on GD

Reply:
ooooh I understand. that change is pointless as the env is in the `.env.docker`